src/web-components.js

```diff
@@ -48,6 +48,28 @@ export function isShadowDomSupported() {
 }
 
 /**
+ * Returns `true` if Shadow CSS encapsulation is supported.
+ * @param {!Element} shadowRoot - a shadowRoot element created by createShadowRoot in shadowEmbed.js
+ * @return {boolean}
+ */
+export function isShadowCssSupported(shadowRoot) {
+  if (!isShadowDomSupported()) {
+    return false;
+  }
+
+  // DO NOT SUBMIT
+  //TODO: Find a better way to test CSS encapsulation
+  if (shadowRoot.toString() !== '[object ShadowRoot]') {
```

Comment:
Yes, that makes sense here, I also like the idea of testing if it is 'native', compared to simply testing the `shadowRoot.toString`, as that can be easily polyfilled.

I Used your suggestions, and it seemed to be working great. I left the console.log for when we state we do not natively support Shadow CSS. I added a DO NOT SUBMIT since it will only be used for testing

Done